asf-event-data/README.md

```diff
@@ -1,26 +1,64 @@
 
-Welcome to the AWS Open Data Bucket hosting Synthetic Aperture Radar (SAR) data products provided by the Alaska Satellite Facility (ASF). This repository is dedicated to offering valuable SAR data corresponding to various disaster events. SAR products for each disaster event are collected with their own prefix, making it easy for users to locate and access relevant data.
+Welcome to the AWS Open Data Bucket hosting Synthetic Aperture Radar (SAR) data products provided by the 
+Alaska Satellite Facility (ASF). This repository is dedicated to offering valuable SAR data supporting responses 
+to a range of disaster events. 
+
+This repository contains a number of different prefixes, some supporting specific response efforts, and others 
+supporting general disaster response services. 
 
 ## Table of Contents
 1. [Introduction](#introduction)
-2. [Disaster Event Prefixes](#disaster-event-prefixes)
-3. [Data Products](#data-products)
-4. [Accessing the Data](#accessing-the-data)
-5. [License](#license)
-6. [Manifest](#manifest)
+2. [Disaster Support Services](#disaster-support-services)
+3. [Disaster Event Prefixes](#disaster-event-prefixes)
+4. [Data Products](#data-products)
+5. [Accessing the Data](#accessing-the-data)
+6. [License](#license)
+7. [Manifest](#manifest)
 
 ## Introduction
 
-Synthetic Aperture Radar (SAR) data is a powerful tool for monitoring and assessing disaster events. The Alaska Satellite Facility (ASF) has curated this collection of SAR data products, making it accessible to the public through this AWS Open Data Bucket. The data covers a range of disaster events, providing valuable insights for researchers, scientists, and emergency response teams.
+Synthetic Aperture Radar (SAR) data is a powerful tool for monitoring and assessing disaster events. 
+The Alaska Satellite Facility (ASF) has curated this collection of SAR data products, making it accessible 
+to the public through this AWS Open Data Bucket. The data covers a range of disaster events, providing valuable insights for researchers, scientists, and emergency response teams.
+
+## Disaster Support Services
+
+Some of the prefixes include content generated in support of ongoing monitoring services. The data can 
+be accessed through interactive web applications, which automatically provide access to new data products as 
+they are added to the bucket. These prefixes include: 
+
+- RTC_services
+- HKHwatermaps
+
+### RTC Services
+
+The contents of the RTC_services prefix are Radiometric Terrain Corrected (RTC) Sentinel-1 products generated 
+to support the [NASA Earth Science Applied Science Disasters](https://appliedsciences.nasa.gov/what-we-do/disasters) 
+program. These RTC products cover the east and west coasts of the United States and the entire state of Alaska, 
+and can be accessed through image services hosted by ASF. This allows the data to be available immediately to 
+disaster response personnel as soon as possible without any additional processing. 
```

Comment:
RTC_services includes these file extensions. Probably worth noting the water extent maps in this paragraph.

```
VH.tif
VV.tif
VV.tif.xml
WM.tif
WM_HAND.tif
dem.tif
rgb.tif
```

Reply:
@asjohnston-asf Ended up revamping this section a fair bit.